linux/README.md

```diff
@@ -178,7 +178,7 @@ Supported platform amd64, arm64, armhf, ppc64le, s390x (s390x is only available
 ### RPM (RedHat and Suse)
 Supported JDK version 8,11,17,18
 
-Supported platform x86_64, aarch64, armv7hl, ppc64le, s390x (s390x is only available for jdk11+)
+Supported platform x86_64, aarch64, armv7hl, ppc64le, s390x (s390x is only available for jdk8+)
```

Comment:
not sure that comment is correct, should this be `>8`?

Reply:
is 8+ == >8 :D 
i think, jdk11+ means, starting from 12 and so on.
at least we do have s390x for jdk11